mesh/mesh.c

```diff
@@ -638,17 +638,16 @@ static void attach_ready_cb(void *user_data, int status, struct mesh_node *node)
 	l_queue_remove(pending_queue, pending_msg);
 }
 
-static struct l_dbus_message *attach_call(struct l_dbus *dbus,
+static struct l_dbus_message *attach(struct l_dbus *dbus,
 						struct l_dbus_message *msg,
+						bool fd,
```

Comment:
Name is misleading, this is not a descriptor but a flag. `use_fd` would be better.